src/Microsoft.DocAsCode.Metadata.ManagedReference/MsBuildEnvironmentScope.cs

```diff
@@ -29,21 +29,6 @@ public MSBuildEnvironmentScope()
 
         private EnvironmentScope GetScope()
         {
-            var vsInstallDirEnv = Environment.GetEnvironmentVariable(VSInstallDirKey);
```

Comment:
why remove these lines? as when user specifies ENV, we don't even need to go through the following check...